blog/migrations/0031_alter_blogpage_body.py

```diff
@@ -0,0 +1,36 @@
+        ),
+        migrations.RunPython(migrate_simple_incidents),
```

Comment:
One thing I think would be nice here is to add a "reverse" function here that does not change the database, but having one allows the migration to be undone (meaning I can run `docker compose exec django python manage.py migrate blog 0030` to revert the DB state to before this migration -- it's definitely useful for testing the code, when you might have to run and re-run the migration a few times, but I think it's a good practice in general "just in case" we might want to do it in another environment).

[Here's some documentation](https://docs.djangoproject.com/en/3.2/ref/migration-operations/#runpython) but essentially what we want is something like:

```python
migrations.RunPython(migrate_simple_incidents, reverse_code=migrations.RunPython.noop),
```